### DISCUSSION TITLE
Silently failing code

### DISCUSSION THREAD

**Discussion by amos-kibet** [3Stones-io/omedis#346] [General]
Such kinds of function calls can fail, and when they do, they don't crash the server or raise an error:
```elixir
Event.update(...)
```

This can take you a while before you understand why the code/tests are not working. You can fix this by pattern-matching on the return value, like so:
```elixir
{:ok, _event} = Event.update(...)
```
and when a different result format is returned, the server will crash and and the error raised.

**Comment by fradarr** [comment]
Will take this to the wiki, I saw somewhere in a discussion that it came very handy to Almir for the test i belive. Thanks Amos!

**Comment by amos-kibet** [comment]
I made an Elixir drop out of it, see here: https://elixirdrops.net/d/NqDzV2rM
